website/splash_table.py

```diff
@@ -53,6 +63,29 @@ def make_table(results: list):
     return cds, dt
 
 
+def querying(qrystr: str = 'notanobject'):
+    """
+    Queries the database for an object
+
+    Parameters
+    ----------
+    qrystr
+        The partial string of an object
+
+    Returns
+    -------
+    The list of tuples of object info
+    """
+    qry = Qry(qrystr)
+    db = load_db()  # load the database
+    results = db.search_object(qrystr)
+    if qry.default:
+        return results
+    else:
+        results = [obj[:3] for obj in results]
```

Comment:
cuts to name, ra, dec

Reply:
Remember that you can request search_object to return data in a different format, in case astropy Tables or pandas Dataframes would be easier for you to work with.

Also, this comment 'cuts to name, ra, dec' should be explicitly in the code